notebooks/06_spatial_autocorrelation.md

```diff
@@ -310,10 +310,8 @@ where $n$ is the  number of observations, $z_{i}$ is the standardized value of t
 In order to understand the intuition behind its math, it is useful to begin with a graphical interpretation: the Moran Plot. The Moran Plot is a way of visualizing a spatial dataset to explore the nature and strength of spatial autocorrelation. It is essentially a traditional scatter plot in which the variable of interest is displayed against its *spatial lag*. In order to be able to interpret values as above or below the mean, and their quantities in terms of standard deviations, the variable of interest is usually standardized by subtracting its mean and dividing it by its standard deviation:
 
 ```python
-db['Pct_Leave_std'] = ( db['Pct_Leave'] - db['Pct_Leave'].mean() )\
-                    / db['Pct_Leave'].std()
-db['Pct_Leave_lag_std'] = ( db['Pct_Leave_lag'] - db['Pct_Leave_lag'].mean() )\
-                    / db['Pct_Leave_lag'].std()
+db['Pct_Leave_std'] = ( db['Pct_Leave'] - db['Pct_Leave'].mean() )
+db['Pct_Leave_lag_std'] = ( db['Pct_Leave_lag'] - db['Pct_Leave_lag'].mean() )
```

Comment:
If we remove the standardization in the code, we need to remove it from the text in 310